_posts/events/2019-08-02-mwe.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+categories: [event]
+event_name: "Joint Workshop on Multiword Expressions and WordNet (MWE-WN 2019)"
+event_url: "http://multiword.sourceforge.net/mwewn2019"
+event_location: "Florence, Italy"
+event_submission_date: "2019-05-01"
+host_name: "ACL 2019"
+host_url: "http://www.acl2019.org/"
+siglex_endorsed: yes
+---
+Joint Workshop on Multiword Expressions and WordNet (MWE-WN 2019)
+
+ACL 2019 (Florence, Italy), August 2nd, 2019  
+<http://multiword.sourceforge.net/mwewn2019>  
```

Comment:
Maybe you want another blank line after the link?

Reply:
Done